### PR TITLE
feat: Flyway Migration - Create All Tables

### PR DESCRIPTION
**Pull request by Miaritiana**
@Sanmandresy

### CODE REVIEW

File: src/main/resources/db/migration/V0_2__create_region_table.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+create extension if not exists "uuid-ossp";
```
Comment (Sanmandresy) [inline src/main/resources/db/migration/V0_2__create_region_table.sql]: No need to repeat this here as it was already declared in the first migration script

File: src/main/resources/db/migration/V0_3__create_recipe_table.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+create extension if not exists "uuid-ossp";
```
Comment (Sanmandresy) [inline src/main/resources/db/migration/V0_3__create_recipe_table.sql]: Same

File: src/main/resources/db/migration/V0_4__create_ingredient_table.sql [hunk 3]
```diff
@@ -0,0 +1,6 @@
+create extension if not exists "uuid-ossp";
```
Comment (Sanmandresy) [inline src/main/resources/db/migration/V0_4__create_ingredient_table.sql]: .

File: src/main/resources/db/migration/V0_5__create_allergy_table.sql [hunk 4]
```diff
@@ -0,0 +1,7 @@
+create extension if not exists "uuid-ossp";
```
Comment (Sanmandresy) [inline src/main/resources/db/migration/V0_5__create_allergy_table.sql]: .

File: src/main/resources/db/migration/V0_6__create_recipe_ingredients_table.sql [hunk 5]
```diff
@@ -0,0 +1,7 @@
+create extension if not exists "uuid-ossp";
```
Comment (Sanmandresy) [inline src/main/resources/db/migration/V0_6__create_recipe_ingredients_table.sql]: .

File: src/main/resources/db/migration/V0_7__create_meal_table.sql [hunk 6]
```diff
@@ -0,0 +1,10 @@
+create extension if not exists "uuid-ossp";
```
Comment (Sanmandresy) [inline src/main/resources/db/migration/V0_7__create_meal_table.sql]: .

File: src/main/resources/db/migration/V0_7__create_meal_table.sql [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+
+CREATE TABLE IF NOT EXISTS "recipe_ingredients"(
```
Comment (Sanmandresy) [inline src/main/resources/db/migration/V0_7__create_meal_table.sql]: why are you creating "recipe_ingredients" again ?

**Review by Sanmandresy — CHANGES_REQUESTED** [review]

File: src/main/resources/db/migration/V0_7__create_meal_table.sql (hunk 7 above)
Comment (Miaritiana) [reply src/main/resources/db/migration/V0_7__create_meal_table.sql]: Oh! Excuse me.

**Review by Sanmandresy — APPROVED** [review]

**Pull request merged**